### ISSUE TITLE
Proposal to add `FoodPantry` as type (or provide help to identify the preferred type)

### ISSUE THREAD

**Issue by supersistence** [schemaorg/schemaorg#3143]
Critical social safety net providers do not have an industry standard for operation and hours formats. This creates additional friction for potential client seeking their support.  Working with a network of providers towards collating operations and hours data, and am hoping to see a new `type` established or help to identify the preferred formatting.

**Comment by Aaranged** [comment]
This seems obviously useful, and could be easily accommodated under LocalBusiness (rather than NGO to access openingHours, per AnimalShelter). While I appreciate [there's a difference](https://blog.feedingwestchester.org/blog/3-ways-to-distinguish-a-food-bank-from-a-food-pantry) between food banks and food pantries, I think the former is [more ubiquitous](https://trends.google.com/trends/explore?geo=CA&q=food%20bank,food%20pantry).

This more broadly raises the question of vocabulary for social services. This was raised by @thadguidry [many moons ago](https://github.com/schemaorg/schemaorg/issues/425), and I think @supersistence makes a good case for revisiting this domain: organizations like food banks should have a mechanism to digitally assert the services they offer, where they can be found, and their hours of operation.

**Comment by thadguidry** [comment]
Luckily, Yelp took my call to action many moons ago:  https://www.yelp.com/nearme/community-service-non-profit
But having a designated type as mentioned in #425 is still needed to more broadly cover any social service/community service...whatever we ultimately decide to call that parent type.

**Comment by supersistence** [comment]
Glad there is interest, and excited to see community services easily findable on Yelp!

**Tl;dr there are far more food pantries than food banks.**
On the terminology front, there are indeed differences between food banks and pantries. As @Aaranged link notes, the food banks are the warehouse and often provide other wrap-around services (e.g., SNAP enrollment support). However in the distribution approach applied by Feeding America partners, food banks are the hub while pantries are the spokes (termed partner agencies in food bank parlance). A single food bank will often have dozens to hundreds of partner agencies that provide various food outreach (say soup kitchen or pantry or after-school snacks). There is general food pantry type taxonomy, but it varies between from a bank to bank. As a mechanism to increase *client* access however I believe food pantry is the most widely recognized term, hence my suggestion. In Hawaii for example we have a handful of food banks and hundreds of food pantries. Not a perfect catch all but as many food bank warehouses *also* provide direct service (i.e., give food to people) the food pantry term may provide the most coverage.

**Comment by greggish** [comment]
Hi folks, great to see discussion about this topic. 

Fortunately there is in fact an industry standard for social service operation hours – [Open Referral](@openreferral)'s [Human Service Data Specification](http://docs.openreferral.org/), which has been endorsed by the Alliance of Information and Referral Systems (the industry association for social service referral providers in the U.S.) and is [widely used in Canada and the UK as well](https://openreferral.org/category/blog/). 

Last year, we finished [a successful pilot of alignment of HSDS with schema.org tags](https://openreferral.org/tag/schema-org/). It's on our roadmap this year to refactor the mappings from that pilot into a general purpose proposal that we can share here, and I'd really welcome an opportunity to discuss with y'all.

There is also, separately, an industry standard for classification of service `type` – the [211Taxonomy](http://211taxonomy.org/). We're exploring opportunities to make that easier to use in ways like this, and I'd be glad to discuss. Would strongly recommend not starting from scratch, as many have tried and failed (and the few who really made a go of it have made the problem much worse).

**Comment by thadguidry** [comment]
@greggish nice!  I looked in the repo at the [hsds-transformer](https://github.com/openreferral/hsds-transformer) but didn't see the mappings by the Stanford team.  Would be good to share link to the mappings so everyone can start reviewing and probably best for you to start a GitHub discussion for collaboration, then we can circle back to this narrow issue on FoodPantry services.

**Comment by greggish** [comment]
hrm, this is all from a year ago and I'm excited to start the next cycle on the project but we're not there yet... for now I think we can link to [this mapping spreadsheet](https://docs.google.com/spreadsheets/d/1YhsKoLz_sz1a4NV-51WgnL8dRWvYjt3-_H4-dGQ54tQ/edit#gid=1793341370) which was our main point of reference for the project though a fair amount of context may be important for that to make sense. 

Is there an active initiative on the schema.org side pertaining to social services that we should know about? If so, I'd be glad to learn more and get involved. If not, we'll have a proposal that we're excited to bring forth... just timeline tbd!

**Comment by thadguidry** [comment]
Nothing active for social services, because Service and its available properties fits the 90% of the need out there.  It's the gaps from community interest that we are attempting to gather together and then mint new properties that might be particularly important and not just my proposed Social Service type.  I.E. It's the properties that we might be missing under Service that any Social Service would say is particularly important and missing currently.  So any insight you can give us on those would be "swell" 😉

**Comment by greggish** [comment]
IIRC most of what is in HSDS can also be found in schema.org, except for [`status` in `service`](http://docs.openreferral.org/en/latest/hsds/reference/#service) which is used for something different in schema.org

I think the tricky part is going to be mapping the relationships carefully because services tend to be more complex and variable (n:m relationships between services and locations, with service info that varies across locations). 

@bjwebb might have other insights to share... although this is something we've tabled to revisit after a currently-underway upgrade for HSDS...

**Comment by Bjwebb** [comment]
I think you've covered the main points.

> IIRC most of what is in HSDS can also be found in schema.org, except for [`status` in `service`](http://docs.openreferral.org/en/latest/hsds/reference/#service) which is used for something different in schema.org

I think in the work so far I was checking **required** fields in HSDS can be found in schema.org (e.g. `status` in `service` is a required HSDS field missing from schema.org). I think there's also a few optional HSDS fields which aren't in schema.org (e.g. [`tax_status` in `organization`](http://docs.openreferral.org/en/latest/hsds/reference/#organization)).
